tests/test_stump.py

```diff
@@ -17,19 +17,8 @@
     ),
 ]
 
-
-@pytest.mark.parametrize("Q, T", test_data)
-def test_calculate_squared_distance_profile(Q, T):
```

Comment:
I think this test should be moved to `test_core.py`

Reply:
Maybe I'm just not seeing `test_core.py` as a changed file but the test coverage should've brought it up as a function that hasn't been tested